packages/projection-typeorm/src/entity/GovernanceAction.entity.ts

```diff
@@ -0,0 +1,39 @@
+import { BigIntColumnOptions, OnDeleteCascadeRelationOptions } from './util';
+import { BlockEntity } from './Block.entity';
+import { Cardano } from '@cardano-sdk/core';
+import { Column, Entity, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from 'typeorm';
+import { Hash32ByteBase16 } from '@cardano-sdk/crypto';
+
+@Entity()
+export class GovernanceActionEntity {
+  @PrimaryGeneratedColumn()
+  id?: number;
+
+  // LW-11270
+  // This is required to handle rollbacks, once we have transactions projection
+  // the OnDeleteCascadeRelationOptions can be moved on txId and this column could be removed
+  @ManyToOne(() => BlockEntity, OnDeleteCascadeRelationOptions)
+  @JoinColumn()
+  block?: BlockEntity;
+
+  @Column('varchar')
+  txId?: Cardano.TransactionId;
+
+  @Column('smallint')
+  index?: number;
+
+  @Column('varchar')
+  rewardAccount?: Cardano.RewardAccount;
+
+  @Column('varchar')
+  anchorUrl?: string;
+
+  @Column('char', { length: 64, nullable: true })
+  anchorHash?: Hash32ByteBase16;
+
+  @Column(BigIntColumnOptions)
+  deposit?: bigint;
+
+  @Column('varchar')
+  action?: string;
```

Comment:
Can we create a better type for this? What's the json structure?
```suggestion
  @Column('varchar')
  action?: Cardano.GovernanceAction;
```

Reply:
ded3e97e6d1 should solve this